addons/wp/functions/fnc_findClosestTarget.sqf

```diff
@@ -16,13 +16,21 @@
  * Public: No
 */
 
-params ["_group", ["_radius", 500]];
+params ["_group", ["_radius", 500], "_area"];
```

Comment:
Not really sure about the use for `_area`
wouldn't this be better
`params ["_group", ["_radius", 500], ["_area", [], [[]], [4]] ];`

This check it with `if !(_area isEqualTo []) then {`